.azure/templates/jobs/build/release_artifacts.yaml

```diff
@@ -11,13 +11,21 @@ jobs:
       # Install Prerequisites
       - template: "../../steps/prerequisites/install_yq.yaml"
       - template: "../../steps/prerequisites/install_asciidoctor.yaml"
+      - template: '../../steps/prerequisites/install_java.yaml'
 
       # Build the release files
       - bash: "make release"
         displayName: "Build release files"
         env:
           RELEASE_VERSION: '${{ parameters.releaseVersion }}'
 
+      # Build Java code with changed release version
```

Comment:
Where exactly do you change the version?

Reply:
it's part of the `make release` goal, where the version in the Maven pom.xmls is set to the one specified by `RELEASE_VERSION`.
My thinking is that it will be still there in the next step.